composeApp/src/commonMain/kotlin/com/github/familyvault/ui/screens/main/modifyFamilyMember/ModifyFamilyMemberScreen.kt

```diff
@@ -0,0 +1,121 @@
+package com.github.familyvault.ui.screens.main.modifyFamilyMember
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.SegmentedButton
+import androidx.compose.material3.SegmentedButtonDefaults
+import androidx.compose.material3.SingleChoiceSegmentedButtonRow
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import cafe.adriel.voyager.core.screen.Screen
+import cafe.adriel.voyager.navigator.LocalNavigator
+import cafe.adriel.voyager.navigator.currentOrThrow
+import com.github.familyvault.services.IFamilyGroupService
+import com.github.familyvault.ui.components.InitialScreenButton
+import com.github.familyvault.ui.components.overrides.TopAppBar
+import com.github.familyvault.ui.components.typography.Paragraph
+import com.github.familyvault.ui.theme.AdditionalTheme
+import familyvault.composeapp.generated.resources.Res
+import familyvault.composeapp.generated.resources.user_modification_choose_permission_content
+import familyvault.composeapp.generated.resources.user_modification_guardian
+import familyvault.composeapp.generated.resources.user_modification_guest
+import familyvault.composeapp.generated.resources.user_modification_member
+import familyvault.composeapp.generated.resources.user_modification_save_button
+import kotlinx.coroutines.launch
+import org.jetbrains.compose.resources.stringResource
+import org.koin.compose.koinInject
+
+class ModifyFamilyMemberScreen(val userId: String) : Screen {
+    @Composable
+    override fun Content() {
+        val navigator = LocalNavigator.currentOrThrow
+        val options = listOf(
+            stringResource(Res.string.user_modification_guest),
+            stringResource(Res.string.user_modification_member),
+            stringResource(Res.string.user_modification_guardian)
+        )
```

Comment:
Może to iterować po jakimś modelu stricte UI'owym gdzie jest nazwa i enum z tą grupą uprawnień? Wtedy byś załatwił sprawe mapowania i wyrzucił `forEachIndexed`